### PR TITLE
Warmup yql

### PR DESCRIPTION
**Pull request by baldersheim**
@bratseth PR

### CODE REVIEW

File: container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java [hunk 1]
```diff
@@ -35,44 +40,61 @@
...
+        this(new SimpleLinguistics());
+    }
+    static boolean warmup() {
+        return warmup(new SimpleLinguistics());
+    }
+    static boolean warmup(Linguistics linguistics) {
+        Query query = new Query("search/?yql=select%20*%20from%20sources%20where%20title%20contains%20'xyz';");
+        Result result = insertQuery(query, new ParserEnvironment().setLinguistics(linguistics));
+        if (result != null) {
+            log.warning("Something fishy. Reult = " + result.toString());
```
Comment (bratseth) [inline container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java]: "Result". But this message is in itself quite fishy. Could we make the error conditions here more crisp?

Comment (kkraune) [inline container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java]: Result

File: container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java [hunk 2]
```diff
@@ -35,44 +40,61 @@
...
+    public MinimalQueryInserter(Linguistics linguistics) {
+        warmup(linguistics);
+    }
+    MinimalQueryInserter() {
+        this(new SimpleLinguistics());
+    }
+    static boolean warmup() {
+        return warmup(new SimpleLinguistics());
+    }
+    static boolean warmup(Linguistics linguistics) {
```
Comment (bratseth) [inline container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java]: Could you add a comment which explains why we want to do warmup here?

File: container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java (hunk 1 above)
Comment (baldersheim) [reply container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java]: Fixed

File: container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java (hunk 2 above)
Comment (baldersheim) [reply container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java]: Fixed

File: container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java (hunk 1 above)
Comment (baldersheim) [reply container-search/src/main/java/com/yahoo/search/yql/MinimalQueryInserter.java]: Crisper..

**Pull request merged**